### PR TITLE
add feature1

### PR DESCRIPTION
**Pull request by mantisa1980**
please review my code and merge feature1

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -2,3 +2,6 @@ class Foo(object):
 	def __init__(self):
 		self.x=1
 		self.y=3 # by duyhsieh
+
+	def feature1(self):
```
Comment (igs-duyhsieh) [inline src/main.py:6]: what is the feature about ? nothing is inside and no comment at all.

Comment (mantisa1980) [reply src/main.py:6]: this is a test function only. no harm.

**Review by igs-duyhsieh — COMMENTED** [review]
my review is done . but not yet approved

**Review by igs-duyhsieh — CHANGES_REQUESTED** [review]
add some code in your feature1 func

**Review by igs-duyhsieh — APPROVED** [review]

**Review by igs-duyhsieh — APPROVED** [review]

**Pull request merged**